tests/FSharpPlus.Tests/Task.fs

```diff
@@ -828,7 +828,7 @@ module Task =
 
         // no need to call this, we just want to check that it compiles w/o warnings
         let testDefaultInferenceForReturnFrom() =
-            let t: Task<_> = monad' { return Some "x" }
+            let t: Task<string option> = monad' { return Some "x" }
```

Comment:
This pops out

Reply:
This is a regression. I posted a minimized repro here: https://github.com/dotnet/fsharp/issues/12929